cookbooks/munin/files/default/rrddump.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+RRD_DIR=/var/lib/munin/openstreetmap
+DIR=`mktemp -d`
+NPROCS=8
```

Comment:
What's the advantage of this? It doesn't seem like the time taken to do the dump is important and as CPU is the most critical resource on the munin server I'd rather minimise how much the dump uses.